src/main/java/frc/robot/subsystems/NewClimber.java

```diff
@@ -155,26 +172,61 @@ public void traverseclimberIn(double speed) {
             setTraverseMotorSpeed(0);
             return;
         }
-
         setTraverseMotorSpeed(speed);
     }
```

Comment:
@veggie2u Noticed that these are duplicate implementations of `midClimberArmUp` and `traverseClimberArmUp` respectively. Is there one we would prefer to keep?